### PR TITLE
♻️ Rewrite ampbyexample.com links to internal ones.

### PR DESCRIPTION
**Pull request by robinvanopstal**

### CODE REVIEW

File: pages/content/amp-dev/documentation/guides-and-tutorials/develop/live_blog.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ You can add the disabled attribute to  the hosted page to prevent the polling me
...
+When you publish a blog post, it’s important to be able to deep link to the post to enable features like sharing. With [`amp-live-list`]({{g.doc('/content/amp-dev/documentation/components/reference/amp-live-list.md', locale=doc.locale).url.path}}), deep linking is possible by simply using the `id` of the blog item. For example, [https://amp.dev/documentation/examples/news-publishing/live_blog/index.html#post3]({{g.doc('/content/amp-dev/documentation/examples/documentation/Live_Blog.html', locale=doc.locale).url.path}}#post3)({{g.doc('/content/amp-dev/documentation/examples/documentation/Live_Blog.html', locale=doc.locale).url.path}}#post3) allows you to navigate directly to the blog post with the `post3` id.
```
Comment (matthiasrohmer) [inline pages/content/amp-dev/documentation/guides-and-tutorials/develop/live_blog.md]: This should go to the preview: `{{ g.doc('/content/amp-dev/documentation/examples/previews/Live_Blog.html', locale=doc.locale) }}#post3`

File: pages/content/amp-dev/documentation/guides-and-tutorials/develop/login_requiring/login.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ $order: 1
 description: 'The first time you land on the page, you can see 2 comments and a login button. If you look for the login button in the code, you will find ...'
 ---
 
-The first time you land on the [page](https://ampbyexample.com/samples_templates/comment_section/preview/), you can see 2 comments and a login button.
+The first time you land on the [page]({{g.doc('/content/amp-dev/documentation/examples/documentation/Comment_Section.html', locale=doc.locale).url.path}}), you can see 2 comments and a login button.
```
Comment (matthiasrohmer) [inline pages/content/amp-dev/documentation/guides-and-tutorials/develop/login_requiring/login.md]: This should go to the preview.

Comment (robinvanopstal) [reply pages/content/amp-dev/documentation/guides-and-tutorials/develop/login_requiring/login.md]: Done, now linked to: {{g.doc('/content/amp-dev/documentation/examples/interactivity-dynamic-content/comment_section/preview/index.html', locale=doc.locale).url.path}}

File: pages/content/amp-dev/documentation/guides-and-tutorials/develop/live_blog.md (hunk 1 above)
Comment (robinvanopstal) [reply pages/content/amp-dev/documentation/guides-and-tutorials/develop/live_blog.md]: Done

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/optimize_amp.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ For example, by applying the following optimization techniques:
 
 the ["The Scenic" AMPStart template](https://www.ampstart.com/templates) loads [two seconds faster on a 3G connection](https://www.webpagetest.org/video/compare.php?tests=180529_RY_9198dcdba1824c169887c6e40c221dae-r:1-c:0).
 
-If you want to skip the details, check out the [AMP Boilerplate generator](https://ampbyexample.com/boilerplate), which you can use to generate custom optimized AMP pages.
+If you want to skip the details, check out the [AMP Boilerplate generator](https://amp.dev/boilerplate), which you can use to generate custom optimized AMP pages.
```
Comment (matthiasrohmer) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/optimize_amp.md]: Make this just point to `/boilerplate` to be environment agnostic.

File: pages/content/amp-dev/documentation/guides-and-tutorials/contribute/index@ko.md [hunk 4]
```diff
@@ -31,8 +31,8 @@ AMP 프로젝트는 기술적 기여를 강하게 장려합니다.
 [ampproject.org](https://ampproject.org)은 대부분의 사람들이 AMP에 익숙해지는 곳입니다.
 [ampproject 문서 GitHub 프로젝트](https://github.com/ampproject/docs)에서 문서에 기여할 수 있습니다. (심지어 [여러분이 지금 읽고있는 페이지](https://github.com/ampproject/docs/blob/master/content/docs/contribute/contribute.md)를 더 좋게 할 수도 있습니다)
 
-[ampbyexample.com](https://ampbyexample.com)은 어떻게 AMP를 사용하는 지 예제를 제공합니다.
+[examples]({{g.doc('/content/amp-dev/documentation/examples/index.html', locale=doc.locale).url.path}})은 어떻게 AMP를 사용하는 지 예제를 제공합니다.
```
Comment (matthiasrohmer) [inline pages/content/amp-dev/documentation/guides-and-tutorials/contribute/index@ko.md]: Uppercase "examples".

File: pages/content/amp-dev/documentation/guides-and-tutorials/develop/live_blog@id.md [hunk 5]
```diff
@@ -78,7 +78,7 @@ Anda dapat menambahkan atribut yang dinonaktifkan ke halaman yang dihosting untu
...
+Saat Anda mempublikasikan entri blog, penting untuk menempatkan deep link ke postingan untuk mengaktifkan fitur seperti berbagi. Dengan [`amp-live-list`]({{g.doc('/content/amp-dev/documentation/components/reference/amp-live-list.md', locale=doc.locale).url.path}}), deep linking dapat dilakukan dengan hanya menggunakan` id` dari item blog. Misalnya, [https://amp.dev/documentation/examples/news-publishing/live_blog/preview/index.html#post3]({{g.doc('/content/amp-dev/documentation/examples/previews/Live_Blog.html', locale=doc.locale).url.path}}#post3) mengizinkan Anda membuka postingan blog secara langsung dengan ID `post3`.
 
 AMP By Example menggunakan cookie di [contoh blog langsung]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/develop/live_blog.md', locale=doc.locale).url.path}}) untuk membuat konten terkini. Karena itu, jika ini pertama kalinya Anda membuka halaman, postingan dengan ID “post3” mungkin tidak tersedia, dan Anda akan diarahkan ke postingan pertama.
```
Comment (matthiasrohmer) [inline pages/content/amp-dev/documentation/guides-and-tutorials/develop/live_blog@id.md]: Please check for all occurences of "AMP By Example" as well while we are at it!

Comment (matthiasrohmer) [reply pages/content/amp-dev/documentation/guides-and-tutorials/develop/live_blog@id.md]: @sebastianbenz, what do you think? I remember in our call we quickly were on the same page rewriting all references to AMP by Example but @robinvanopstal just figured out that might not be too easy as just replacing "AMP by Example" with "Examples" will sometimes be double as in "see examples at ~~AMP by Example~~Examples". Not to mention i18n.

Comment (sebastianbenz) [reply pages/content/amp-dev/documentation/guides-and-tutorials/develop/live_blog@id.md]: Yes, these need to be manually checked and rephrased.

Comment (matthiasrohmer) [reply pages/content/amp-dev/documentation/guides-and-tutorials/develop/live_blog@id.md]: ... and leave the translations dirty for now?

File: pages/content/amp-dev/documentation/guides-and-tutorials/start/create/basic_markup.md [hunk 6]
```diff
@@ -43,7 +43,7 @@ The content in the body, so far, is pretty straightforward. But there’s a lot
 Use HTTPS: When creating AMP pages and content, you should strongly consider using the HTTPS protocol (vs. HTTP). Although, HTTPS is not required for the AMP document itself or for images and fonts, there are many AMP features that require HTTPS (e.g., video, iframes, and more). To ensure your AMP pages take full advantage of all AMP features, use the HTTPS protocol.  You can learn more about HTTPS in ["Why HTTPS Matters"](https://developers.google.com/web/fundamentals/security/encrypt-in-transit/why-https).
 
 [tip type="tip"]
-Use the [AMP Boilerplate Generator](https://ampbyexample.com/boilerplate/) to quickly get started with creating new AMP pages.
+Use the [AMP Boilerplate Generator](http://amp.dev/boilerplate) to quickly get started with creating new AMP pages.
```
Comment (matthiasrohmer) [inline pages/content/amp-dev/documentation/guides-and-tutorials/start/create/basic_markup.md]: Just `/boilerplate`.

**Pull request merged**